html/changelogs/ianshield.yml

```diff
@@ -0,0 +1,8 @@
+
+author: Ralta
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added ability for Ian to wear Blueshield/CentComm berets."
```

Comment:
Don't do this; we haven't done changelogs like this in years.

Reply:
Dangit, there was me thinking I was following the correct process! :joy:

Reply:
It's okay, I did it too a long time ago. Almost got away with it too :P